packages/models-library/src/models_library/services_resources.py

```diff
@@ -33,13 +33,14 @@
 CPU_100_PERCENT: Final[int] = int(1 * GIGA)
 
 
-class ResourceValue(BaseModel):
+class ResourceValue(BaseModel, validate_assignment=True):
     limit: StrictInt | StrictFloat | str
     reservation: StrictInt | StrictFloat | str
 
     @model_validator(mode="before")
     @classmethod
     def _ensure_limits_are_equal_or_above_reservations(cls, values):
+        # WARNING: this does not validate ON-ASSIGNMENT!
```

Comment:
so how does it work now? what does validate_assignment=True do? nothing?

Reply:
So on assignment it does not work. What I did is recreating everytime the model. It has only two values so it is not that heavy